root/edit/details/AddStandaloneRecording.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * @flow
+ * Copyright (C) 2020 Anirudh Jain
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import EntityLink from '../../static/scripts/common/components/EntityLink';
+import ExpandedArtistCredit from
+  '../../static/scripts/common/components/ExpandedArtistCredit';
+import formatTrackLength from
+  '../../static/scripts/common/utility/formatTrackLength';
+import yesNo from '../../static/scripts/common/utility/yesNo';
+
+type AddStandaloneRecordingProps = {
+  +display_data: {
+    +artist_credit: ArtistCreditT,
+    +comment: string | null,
+    +length: number | null,
+    +name: string,
+    +recording: RecordingT,
+    +video: boolean,
+  },
+};
+
+const AddStandaloneRecording = (
+  {edit}: {+edit: AddStandaloneRecordingProps},
+) => {
+  const display = edit.display_data;
+  return (
+    <>
+      <table className="details">
+        <tbody>
+          <tr>
+            <th>{addColonText(l('Recording'))}</th>
+            <td><EntityLink allowNew entity={display.recording} /></td>
+          </tr>
+        </tbody>
+      </table>
+      <table className="details add-standalone-recording">
+        <tbody>
+          <tr>
+            <th>{addColonText(l('Name'))}</th>
+            <td>
+              <EntityLink
+                allowNew
+                content={display.name}
+                entity={display.recording}
+                showDisambiguation={false}
+              />
```

Comment:
I would change this to just print the name, same as how it's done in Add release. I see no reason to link the same recording twice. @mwiencek, @yvanzo: opinions?

Reply:
It was this way according to the previous template... and yes I do thing using links twice is not of much use 🤔. It mostly depends on what experience do you want to give

Reply:
I would prefer dropping the `Recording:` row if that is possible, see the above https://github.com/metabrainz/musicbrainz-server/pull/1500#discussion_r422478389.

Reply:
+1 to only printing the name.